assets/js/app.js

```diff
@@ -0,0 +1,54 @@
+function collectData() {
+    const productName = document.getElementById('productName').value
+    const price = document.getElementById('price').value
+    const productCategory = document.getElementById('productCategory').value
+    const imageUrl = document.getElementById('imageUrl').value
+    return {
+        productName: productName,
+        price:price,
+        productCategory: productCategory,
+        imageUrl:imageUrl
+    }   
+}
+
+function generateHTML(data) {
+    const newHTML = `
+        <tr>
+            <td>${data.productName}</td>
+             <td>${data.price}</td>
+              <td>${data.productCategory}</td>
+             <td><img src="${data.imageUrl}" class="color-cell" /></td>
```

Comment:
I'd choose a better more meaningful meaning than `color-cell`. I assume it's a copy-paste - nevertheless...